src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleEntries = sourceString.split(';');
+
+  const stylesObj = styleEntries.reduce((prev, style) => {
+    const [key, value] = style.split(':', 2);
```

Comment:
When using `split`, be aware that if there are more than one colon in the style declaration, it will only split at the first colon. This behavior is correct for CSS parsing, but it's worth noting in case the input string could contain unexpected formats.